ores/scoring_systems/celery_queue.py

```diff
@@ -23,13 +23,14 @@
 
 
 @before_task_publish.connect
-def update_sent_state(sender=None, body=None, **kwargs):
+def update_sent_state(sender=None, body=None, properties=None, **kwargs):
     for application in _applications:
         task = application.tasks.get(sender)
         backend = task.backend if task else application.backend
 
-        logger.debug("Setting state to {0} for {1}".format(SENT, body['id']))
-        backend.store_result(body['id'], result=None, status=SENT)
+        id = properties['correlation_id']
+        logger.debug("Setting state to {0} for {1}".format(SENT, id))
```

Comment:
id is a built-in function in python (=a reserved keyword) per pep8 it should be id_ to avoid clashes.